render.yaml

```diff
@@ -40,6 +45,21 @@ services:
           type: web
           property: host
 
+  - type: web
+    name: mymeta
+    env: node
+    plan: starter
+    buildCommand: yarn && yarn web:build
+    startCommand: yarn --cwd packages/web/ start
+    envVars:
+      - key: NODE_VERSION
+        value: 12.18.3
```

Comment:
I think we just have 12.x defined in the build. Would be good to keep that the same if you can just specify the major version

Reply:
I'll see if I can set it that way.